tests/index.html

```diff
@@ -290,6 +289,15 @@
 		//	Clear the shapes and load the 'arrow' polygon from the physicsData JSON file in the cache
 		contra.body.clearShapes();
 		contra.scale.setTo(arrowProportion, arrowProportion);
+		
+		//add arrow to a group
+		//groupArrow = game.add.group();
+		//groupArrow.add(contra);
+		
+		//Level text
+		var textL = "Level "+speed;
+		var style = { font: "Arial", fontSize: 60*toolbarSz/288, fill: "#ff0044", align: "center" };
```

Comment:
the color of the text should also be a variable that changes according to the "theme" selected, not a fixed value